om-440

sidhartha-citron opened the pull request.

MHigel reviewed (commented):
@sidhartha-citron I'm seeing a lot of changes here unrelated to the requirements in OM-440.  Can you verify that all of the items being removed from this Workflow are correct?

sidhartha-citron commented:
Added all the workflowrules for Opportunity. Before i was adding only the new Workflow rule which got created. Seems it is overriding and removing other rules which it should be appending not removing. Now i have added all the rules.

MHigel commented on force-app/main/default/workflows/Opportunity.workflow-meta.xml:
```diff
@@ -191,21 +202,70 @@
-        <name>Update Account Type</name>
+        <fullName>Update_Op_Name</fullName>
+        <field>Name</field>
+        <formula>Account.Name &amp; &quot; - &quot; &amp;  TEXT(Opportunity_Type__c) &amp; &quot; - &quot; &amp; TEXT(CloseDate)</formula>
```
This commit will revert changes done to the formula that incorporates the `Subsidiary__c` value:

```<formula>AND(   ISPICKVAL(StageName, &apos;Signed&apos;) ,   ISPICKVAL( Processing_Status__c ,&apos;Completed&apos;),ISPICKVAL( Account.Subsidiary__c  , &apos;CA&apos;) )</formula>```

sidhartha-citron replied:
This is fixed now. Added Update Account Type Workflow Rule
